var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -49,15 +50,21 @@ class Adios(AutotoolsPackage):
             description='Enable Fortran bindings support')
 
     variant('mpi', default=True, description='Enable MPI support')
+    variant('no_mpi', default=False, description='Disable MPI support')
```

Comment:
In this case, `+no_mpi` is not the same as `~mpi`. `~mpi` will disable building with a user-specified mpi. adios will still be built with an mpi since it is mainly a parallel library.

`+no_mpi` will explicitly disable mpi and build adios as a seqential library. I have updated the description to better reflect this.